package.json

```diff
@@ -25,6 +25,7 @@
     "generate": "nuxt generate",
     "start": "nuxt start",
     "prod": "pnpm build:only && pnpm start",
+    "prod:playwright": "pnpm i18n:copy-test-locales && pnpm build:only && pnpm start",
```

Comment:
I'm not sure if we actually need a separate `prod:playwright` or if we only use `prod` for playwright tests.
This particular script is necessary to make sure that the locale files used when running `playwright:test` locally, the locale files from `test/locales` are used, not the ones downloaded from `translate.wordpress.org`. CI already does that in `setup-node-env`.

Reply:
Whenever you run `pnpm dev` locally, you load the real locale files from GlotPress. Those files are gitignored in the repo. Then, when you run the playwright tests, those real locale files are used in the Playwright tests, and cause them to error out because we don't really have Arabic translations now (and they are used in the RTL tests).